message-index/messages/GHC-56538/index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Instance head is not headed by a class
+summary: A type class instance declaration is declared for something that is not a type class
+introduced: 9.6.1
+severity: error
+---
+
+Instance declarations specify the operations that place a type into a type class. For example, an instance of `Functor` describes how `fmap` works for a type. Instances only make sense for things that are, in fact, type classes - `instance Int Int` is nonsense, as is `instance thing Int`.
```

Comment:
Please add an example of `Ord` or `Eq`, as a Haskell newbie may have not encountered `Functor` type class, but usually how to overload equality is taught for other languages in CS 101.

Reply:
Great idea! Will do.

Reply:
Done